src/components/HeroProjectContainer.js

```diff
@@ -1,16 +1,33 @@
 import PropTypes from 'prop-types';
 
 function HeroProjectContainer({styles, heroProject}) {
-  const {projectName, images,} = heroProject
+  // eslint-disable-next-line no-unused-vars
+  const {projectName, images, technologiesUsed, links, description} = heroProject
+
+  const tech = technologiesUsed.map((tech, index) => {
+    if (index !== technologiesUsed.length - 1)
+      return `${tech},`
+    else
+      return `${tech}`
+  })
```

Comment:
Hey @jedwardmook , what if you used join here to make a string that is joined with the comma? I think it uses a bit less computing power to calculate this.

```
const tech = technologiesUsed.join(',');
```